packages/browserify/test/fixtures/secureBundling/lavamoat/node/policy.json

```diff
@@ -60,14 +60,10 @@
     "@lavamoat/aa": {
       "builtin": {
         "fs.readFileSync": true,
-        "fs.statSync": true,
         "path.dirname": true,
         "path.join": true,
         "path.relative": true
       },
-      "globals": {
-        "performantResolve": true
```

Comment:
Do you know why this change is here?

Reply:
It looks correct to me.  In fact, running `npm run test:prep` in `main` results in this change.

I think we need to add something to ensure these generated policy files get updated properly, but I'm not sure what that looks like yet.  I'll make an issue for it.